docs/analysis_plan.md

```diff
@@ -0,0 +1,90 @@
+# Analysis plan
+
+This document describes the analysis plan to perform a retrospective evaluation of the impact of incorporating wastewater data on the forecast performance of a semi-mechanistic renewal-based model.
+Forecasts of COVID-19 hospital admissions will be generated weekly and evaluated
+against the final observed hospital admissions data, across all age groups.
+
+# Aims
+
+The goal of these analyses will be to assess:
+1. How does incorporating wastewater impact the forecast performance of the semi-mechanistic renewal model?
+2. How do characteristics of the forecast problem and the wastewater surveillance in that forecast problem impact variation in performance between models with and without wastewater? 
+
+# Data
+
+This analysis will use publicly available data from RKI of COVID-19 hospital admissions at the state-level and wastewater concentration data at the site-level in Germany during the 2024-2025 respiratory virus season.
+We will use GitHub's commit history of both repositories to generate snapshots of each data source available as of each forecast date.
+Weekly snapshots of both datasets will be created every Monday from October 2024 to March 2025.
+
+## Hospital admissions data
+Hospital admissions data are reported by age group and state as 7-day rolling sums.
+We will fit and generate forecasts for all age groups, at the state-level.
+We will take the difference between successive days to compute daily hospital admissions incidence as an input into the model. 
+
+
+### Addressing right-truncation
+The hospital admissions data provided are indexed by the data of the positive test -- which is sometimes well before patients end up hospitalised, leading to a right-truncation problem described in Wolffram et al.
+
+We will address this by separately estimating a delay distribution for each region and forecast date using `baselinenowcast`, and modifying the `wwinference` model to account for right-truncated data via incorporating a reporting delay in the observation model.
+
+## Wastewater concentration data
+Data on wastewater concentrations of SARS-CoV-2 are available from individual wastewater treatment plants (will refer to these as sites), indexed by sample collection date.
+Each wastewater treatment plant is contained within a state, and contains the metadata on the catchment area (population size) served by the wastewater treatment plant.
+We will assume all individuals served by the wastewater treatment plant reside within the state.
+We will jointly fit to all wastewater treatment plants within a state.
```

Comment:
again not part of the model section. I would suggest: data, model specification, model validaiton, how we use the model but snappier title, evaluation, implementation

Reply:
Made a new issue to reorg this